### PR TITLE
protobuf upgrade

### PR DESCRIPTION
**Pull request by mikea**

### CODE REVIEW

File: build/deps/build_deps.jsonc [hunk 1]
```diff
@@ -78,6 +78,22 @@
...
+      "name": "aspect_rules_esbuild",
+      "type": "github_release",
+      "owner": "aspect-build",
+      "repo": "rules_esbuild",
+      "file_regex": "\\.tar\\.gz$",
+      // All releases are marked as prerelease, so github_last_release() fails.
+      "freeze_version": "v0.21.0"
+    },
+    {
+      "name": "rules_pkg",
```
Comment (fhanau) [inline build/deps/build_deps.jsonc:92]: Is rules_pkg needed by protobuf now?

File: build/deps/gen/dep_com_google_protobuf.bzl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# WARNING: THIS FILE IS AUTOGENERATED BY update-deps.py DO NOT EDIT
+
+load("@//:build/http.bzl", "http_archive")
+
+TAG_NAME = "v28.2"
```
Comment (fhanau) [inline build/deps/gen/dep_com_google_protobuf.bzl:5]: This puts us far ahead of the protobuf version used in perfetto, V8 and the downstream repo, but the build seems to work so maybe that's just fine?

**Review by fhanau — APPROVED** [review]

File: build/deps/build_deps.jsonc (hunk 1 above)
Comment (mikea) [reply build/deps/build_deps.jsonc:92]: yes

File: build/deps/gen/dep_com_google_protobuf.bzl (hunk 2 above)
Comment (mikea) [reply build/deps/gen/dep_com_google_protobuf.bzl:5]: I guess so. Looks like it is version-compatible?

**Review by anonrig — APPROVED** [review]

**Pull request merged**